Add an animation for HLD (#3732)
cpinitiative/usaco-guide: content/5_Plat/HLD.mdx

to node $y$ will take $\mathcal{O}(\log^2 N + \log N)$ = $\mathcal{O}(\log^2 N)$
time. We can answer $Q$ queries in $\mathcal{O}(Q \log^2 N)$ time.

> Here is an animations of how it works:

Reviewer: ```suggestion
Here's a helpful animation of how the algorithm works:

```

Author: Done
